app/src/main/java/de/moyapro/idle/domain/Biome.kt

```diff
@@ -11,7 +11,7 @@ data class Biome(
 
     fun generate(seconds: Int = 1): Biome {
         species.shuffled().forEach {
-            val generatedResources = it.generationAndComsumption(seconds)
+            val generatedResources = it.generationAndComsumption(seconds, it)
```

Comment:
Sie fressen sich ja doch selber.... So war das nicht gedacht. Hier muss statt it species rein, also alle anderen